pymc/distributions/distribution.py

```diff
@@ -164,6 +165,45 @@ def fn(*args, **kwargs):
     return fn
 
 
+# Helper function from pyprob
```

Comment:
URL to original commit on pyprob: https://github.com/pyprob/pyprob/commit/6aac747a8d9a55ca91d6408fee9a386289720a8e